testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/testresult/CamundaProcessTestResultPrinter.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.process.test.impl.testresult;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+
+public class CamundaProcessTestResultPrinter {
+
+  private static final String NO_ENTRIES = "<None>";
+  private static final int MAX_VALUE_LENGTH = 500;
+
+  private final Consumer<String> printStream;
+
+  public CamundaProcessTestResultPrinter(final Consumer<String> printStream) {
+    this.printStream = printStream;
+  }
+
+  public void print(final ProcessTestResult result) {
+    final String formattedResult = formatResult(result);
+    printStream.accept(formattedResult);
+  }
+
+  private String formatResult(final ProcessTestResult result) {
+    return "Process test results:\n"
```

Comment:
❓ Do you think `StringUtils` can be helpful here ? Improving maintainability.

Reply:
I have no idea how to utilize `StringUtils` here. I'm open to your ideas. :+1:

Reply:
Sorry, I meant `StringBuilder` 🙏